examples/snippets/utils/client/manual_auth.py

```diff
@@ -0,0 +1,21 @@
+"""
+Snippet for creating a tamr client object with manual passing of credentials during runtime.
+
+Note: This approach cannot be coupled with fully automated workflows (e.g., crontab jobs)
+"""
+
+from tamr_toolbox import utils as tbu
+from getpass import getpass
+
+
+# load configuration variables that do not include AUTH credentials
```

Comment:
I'm being very nitpicky here, but it would be nice to get rid of the extra lines and swap the order of imports so `tamr_toolbox` comes after the standard library `getpass`
 
```suggestion
"""
from getpass import getpass
from tamr_toolbox import utils as tbu

# load configuration variables that do not include AUTH credentials
```